### PR TITLE
arm/stm32f401rc-rs485: Add support to RFID MFRC522

### PR DESCRIPTION
**Pull request by rcsim**
## Summary
Add support to RFID MFRC522

## Impact
Users will be able to use MFRC522 with stm32f401rc-rs485:

## Testing
- Configure mfrc522
- Build and Flash
- Run rfid_readuid example

```
nsh> rfid_readuid
Trying to READ: Card is not present!
Trying to READ: Card is not present!
Trying to READ: Card is not present!
Trying to READ: Card is not present!
Trying to READ: RFID CARD UID = 0x3DB3F169
```

### CODE REVIEW

File: boards/arm/stm32/common/src/stm32_mfrc522.c [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+ *
+ ****************************************************************************/
+
+int stm32_mfrc522initialize(const char *devpath)
+{
+  struct spi_dev_s *spi;
+  int ret;
+
+  spi = stm32_spibus_initialize(MFRC522_SPI_PORTNO);
+
```
Comment (acassis) [inline boards/arm/stm32/common/src/stm32_mfrc522.c:69]: Please remove empty line

**Comment by acassis** [comment]
@rcsim please include a Documentation/ with the wiring connection (it is a good idea to include a picture of used module)

File: boards/arm/stm32/common/src/stm32_mfrc522.c (hunk 1 above)
Comment (rcsim) [reply boards/arm/stm32/common/src/stm32_mfrc522.c]: Done, thanks!

**Comment by rcsim** [comment]
> @rcsim please include a Documentation/ with the wiring connection (it is a good idea to include a picture of used module)

Added. Thanks.

**Review by xiaoxiang781216 — APPROVED** [review]

**Review by jerpelea — APPROVED** [review]
please change the title to arm/stm32f401rc-rs485: Add support to RFID MFRC522 

and add a commit message 
Thanks

**Comment by rcsim** [comment]
> please change the title to arm/stm32f401rc-rs485: Add support to RFID MFRC522
> 
> and add a commit message Thanks

I've changed the commit msg as requested. Thanks!

**Review by acassis — APPROVED** [review]

**Pull request merged**